src/components/common/bullets-colin-link.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <div id="bullets-colin-link">
+    <v-row>
+      <v-col cols="12" sm="3">
+        <v-radio-group v-model="selectedCompanyType" @change="radioButtonChanged" flat mandatory>
+          <v-radio id="named-company-radio"
+            class="mb-0 pb-4"
+            :label="getRadioText(CompanyType.NAMED_COMPANY)"
+            :value="CompanyType.NAMED_COMPANY"
+          />
+          <v-radio id="numbered-company-radio"
+            :label="getRadioText(CompanyType.NUMBERED_COMPANY)"
+            :value="CompanyType.NUMBERED_COMPANY"
+          />
+        </v-radio-group>
+      </v-col>
+      <v-col cols="12" sm="9">
+        <div v-if="selectedCompanyType === CompanyType.NAMED_COMPANY">
+          <v-row>
+            <slot name="name-input-slot">Name Input</slot>
+            <template v-if="showDesignation">
+              <slot name="designation">Designation</slot>
+            </template>
+          </v-row>
+        </div>
+        <div v-else>
+          <ul class="bullet-points">
+            <li v-for="bulletPoint in bulletPoints" :key="bulletPoint">
+              {{ bulletPoint }}
+            </li>
+          </ul>
+          <div class="btn-spacing" v-if="colinButton">
+            <v-btn class="px-9" :href="colinLink" target="_blank">
+              Go to Corporate Online to Register <v-icon small class="ml-1">mdi-open-in-new</v-icon>
+            </v-btn>
+          </div>
+          <div class="btn-spacing" v-else>
+            <v-btn
+              class="px-9"
+              @click="incorporateNow"
+            >
+              Incorporate Now
+            </v-btn>
+          </div>
+        </div>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { CompanyType } from '@/enums'
+import NameInput from '@/components/new-request/name-input.vue'
+import { Navigate } from '@/plugins'
+import BusinessServices from '@/services/business.services'
```

Comment:
Can we rename the `xxx.services` files to `xxx-services` (and keep the period for the extension only)?

Reply:
I see you've done this change in your PR Sev. I'll not be making this change in this PR then OK?

Reply:
Yup.